frontend/app/templates/application.hbs

```diff
@@ -1,76 +1,10 @@
-<nav class="navbar navbar-default">
-    <div class="container">
-        <div class="navbar-header">
-            <a class="navbar-brand" href="/">BadgeYAY</a>
+<div class="pusher">
+    {{nav-bar}}
+    <div class="ui container main-container">
+        {{outlet}}
+        <div>
```

Comment:
Why there is empty `div` exist?

Reply:
This will contain other things which will be added by @yashLadha  after this PR gets merged :+1:

Reply:
@yashLadha Outlets are slots in the template where content can be rendered from other routes. It doesn't need an empty div to inflate. Can you please provide resource for the same

Reply:
This has some CSS involved with it as it is quite visible from the fact that a class has been associated with this.

I cannot quite understand your point here 
>It doesn't need an empty div to inflate. Can you please provide resource for the same

It is acting as a container, i.e. anything rendered inside it will have `class=container` on it.
The purpose is quite visible from the line. I do not understand what part are you having problem with?

Reply:
@vaibhavsingh97  The best resource is Ember Documentation but the important part is reading not just pasting here :smile:

Reply:
This was quite an easy thing to understand. Please state your problem with the particular line?

Reply:
I do understand your point. But I was talking about [this ](https://github.com/ParthS007/badgeyay/blob/b655afbdc42a12b377cb81e325b88612236d85a4/frontend/app/templates/application.hbs#L5:L7 )empty division.

Reply:
:man_facepalming: